analysis/analysis-template.yml

```diff
@@ -71,7 +71,7 @@ Resources:
           projection.datetime.format: 'yyyy/MM/dd/HH'
           projection.datetime.interval: 1
           projection.datetime.interval.unit: HOURS
-          storage.location.template: 's3://audit-${Environment}-message-batch/firehose/${datetime}/'

```

Comment:
@Aidan23Smith this needs putting back in but using !Sub as well as per the location below

Reply:
Yes it did